corehq/motech/repeaters/dbaccessors.py

```diff
@@ -122,6 +122,7 @@ def get_paged_couch_repeat_records(domain, skip, limit, repeater_id=None, state=
     kwargs.update(_get_startkey_endkey_all_records(domain, repeater_id, state))
 
     results = RepeatRecord.get_db().view('repeaters/repeat_records', **kwargs).all()
+    results.sort(key=lambda result: (result['doc']['registered_on'] or 0), reverse=True)
```

Comment:
What does `paged` mean in this function name - is this literally pulling a single page in the report? I'd think sorting would need to happen before paging (but then there's the question of performance problems).

Reply:
Good point. I did not do enough local testing to see that this is really just reordering one page of results, not all of them... 
I'm not sure if the right solution is to return all the records, sort and then page them, which I imagine would probably cause performance issues esp. on domains like the one where this issue was noticed, with ~2000 records. The other option seems like it would be modifying the Couch view, since it doesn't seem to have a 'sort' parameter like a SQL query would.